setup.py

```diff
@@ -0,0 +1,34 @@
+import os
+
+import setuptools
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
+with open("requirements.txt") as fh:
+    required = fh.read().splitlines()
+
+with open(os.path.join("ner4opt", "_version.py")) as fp:
+    exec(fp.read())
+
+setuptools.setup(
+    name="ner4opt",
```

Comment:
I assume we will push this to PyPI so that people can do `pip install ner4opt`

Reply:
yes?

Reply:
Yes.